src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -0,0 +1,64 @@
+{% from "./caseStatus.njk" import caseStatusForClaimant, caseStatusForDefendant %}
+{% from "timeRemaining.njk" import timeRemaining %}
+
+{% macro _viewResponseLink (externalId) %}
+  <a href="{{ ResponsePaths.summaryPage.evaluateUri({ externalId: externalId }) }}">{{ t('View full response') }}</a>
+{% endmacro %}
+
+{% macro caseHistoryForClaimant(claim, url) %}
+  {% for state in claim.stateHistory %}
+    <div class="status-block">
+      {% if state.status === ClaimStatus.OFFER_ACCEPTED %}
+        <h3 class="heading-small">{{ t('Settle out of court') }}</h3>
+        <p>
+          {{ t('You’ve agreed to the offer made by {{ defendantName }} and signed an agreement to settle your claim.',
+            { defendantName: claim.claimData.defendant.name }) }}
+        </p>
+        <p>
+          {{ t('We’ve asked {{ defendantName }} to sign the agreement.',
+            { defendantName: claim.claimData.defendant.name }) }}
+        </p>
+      {% elif state.status === ClaimStatus.CLAIM_REJECTED %}
+        <h3 class="heading-small">{{ t('The defendant’s response') }}</h3>
+          <p>
+            {{ t('{{ defendantName }} has rejected your claim. We’ll send you a directions questionnaire – this is a form which you must complete to tell us more about the claim.',
+              { defendantName: claim.claimData.defendant.name }) }}
+          </p>
+          <p>
+            <a href="{{ ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download their response') }}</a>
+          </p>
+      {% else %}
+        {{ caseStatusForClaimant(state.status, claim, url) }}
+      {% endif %}
+    </div>
+  {% endfor %}
+{% endmacro %}
+
+{% macro caseHistoryForDefendant(claim, url, isAfter4pm) %}
+  {% for state in claim.stateHistory %}
+    <div class="status-block">
+      {% if state.status === ClaimStatus.OFFER_ACCEPTED %}
+        <h3 class="heading-small">{{ t('Settle out of court') }}</h3>
+        <p>
+          {{ t('The claimant has accepted your offer. You need to sign a legal agreement to settle out of court. {{ claimantName }} has signed the agreement.',
+            { claimantName: claim.claimData.claimant.name }) }}
+        </p>
+        <p>
+          <a href="{{ OfferPaths.countersignAgreementPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Sign the agreement') }}</a>
+        </p>
+      {% elif state.status === ClaimStatus.CLAIM_REJECTED %}
+        <h3 class="heading-small">{{ t('Your response to the claim') }}</h3>
+        <p>
+          {{ t('You have rejected the claim. The case will be reviewed by a judge and might go to court.') }}
+        </p>
+        <p>
+          <a href="{{ ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId }) }}">{{ t('Download your response') }}</a>
+        </p>
+        {#<h3 class="heading-small">{{ t('Your defence is being processed.') }}</h3>#}
```

Comment:
please remove commented out code
REQ

Reply:
I left it to remember to ask content about it ;)

Reply:
Removed